.snapcraft/resources/restoredb

```diff
@@ -1,72 +1,88 @@
-#! /bin/bash
+#!/bin/bash
 
-if [[ ${EUID} != 0 ]]
-then
+function warn {
+        echo "[!] ${1}"
+        echo "[*] Check ${RESTORE_DIR}/${LOG_NAME} for details."
+}
+
+function abort {
+        echo "[!] ${1}"
+        echo "[*] Check ${RESTORE_DIR}/${LOG_NAME} for details."
+        echo "[-] Restore aborted!"
+        exit
```

Comment:
What about a exit code, like `exit 1`?

Reply:
Yes agreed!  Make clear even if some how in automated fashion or part of another script that it isn't exiting cleanly  :+1: